app/views/hyrax/homepage/_login_modal.html.erb

```diff
@@ -0,0 +1,62 @@
+<%# OVERRIDE FROM HYRAX to restructure add to collection modal #%>
+<div role="dialog" class="modal collection-list-modal fade" id="collection-list-container" tabindex="-1" aria-labelledby="col_add_title" data-backdrop="false">
```

Comment:
Let's use a more unique target/id: `#user-login-modal`

Reply:
Also the `.collection-list-modal` class, is it relevant here and should it be renamed?